QueueManagerTest/QueueFileManagerTests.cs

```diff
@@ -18,14 +25,23 @@ public void SetUp()
             _movieList.Add("movie1");
             _movieList.Add("movie2");
             _queueFileManager = new QueueFileManager();
+            _upperLevelFolderName = @"..\..\Test Files\";
+            _pathString = Path.GetFullPath(_upperLevelFolderName);
+            if (!Directory.Exists(_pathString))
```

Comment:
What comment is that?  Are you asking if it will exist on a fresh machine?  Yes.  I am creating it in case it doesn't exist.